libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2168,6 +2167,9 @@ void TypeChecker::endVisit(NewExpression const& _newExpression)
 		if (!contract->constructorIsPublic())
 			m_errorReporter.typeError(_newExpression.location(), "Contract with internal constructor cannot be created directly.");
 
+		if (contract->abstract())
```

Comment:
The check above (`unimplementedFunctions.empty()`) can now be removed or turned into an assertion below this check.

Reply:
Ok, I removed the check above, but added `&&  !contract-> annotation().unimplementedFunctions.empty()` here.. so we will have the same error. We will later add the information about what functions where not be implemented.